test/unittests/execute_test.cpp

```diff
@@ -1135,6 +1135,50 @@ TEST(execute, imported_functions_call_indirect)
     EXPECT_RESULT(execute(instance, 3, {2, 50}), 7);   // isqrt(50)
 }
 
+TEST(execute, imported_function_from_another_module)
+{
+    /* wat2wasm
+    (module
+      (func $add (param $lhs i32) (param $rhs i32) (result i32)
+        get_local $lhs
+        get_local $rhs
+        i32.add)
```

Comment:
Please use noncommutative operation so we know the order of arguments is correct.

Reply:
Changed to sub